src/main/java/com/exasol/adapter/sql/SqlLimit.java

```diff
@@ -4,25 +4,27 @@
 import com.exasol.adapter.AdapterException;
 
 public class SqlLimit extends SqlNode {
-    
-    int limit;
-    int offset;
-    
+
+    private int limit;
```

Comment:
Add `final`.

Reply:
Can't add final, because setters will be broken